src/mpyl/project.py

```diff
@@ -609,7 +634,9 @@ def load_project(
     with open(root_dir / project_path, encoding="utf-8") as file:
         try:
             start = time.time()
-            yaml_values = YAML(typ=None if safe else "unsafe").load(file)
+            child_yaml_values = YAML(typ=None if safe else "unsafe").load(file)
```

Comment:
nit: keep the old name, you don't know if this is a 'child' yaml 😬

Reply:
[Addresse](https://github.com/Vandebron/mpyl/pull/287/commits/91bb7489d9cfdacc6f7855d6af138c8025478c99)